internal/runbits/dependencies/changesummary.go

```diff
@@ -79,9 +79,8 @@ func OutputChangeSummary(out output.Outputer, newBuildPlan *buildplan.BuildPlan,
 			prefix = "└─"
 		}
 
-		ingredientDeps := ingredient.RuntimeDependencies(true)
 		subdependencies := ""
-		if numSubs := len(ingredientDeps); numSubs > 0 {
+		if numSubs := len(ingredient.RuntimeDependencies(false)); numSubs > 0 {
```

Comment:
We want to count the total number of sub-dependencies, not just direct ones.

The number I see as a user should tell me how many total dependencies this one dependency is bringing in.

Reply:
That's what I thought too, but if we include recursive dependencies, then all sub-dependencies end up having the exact same, 200+ dependency number.